src/cli.js

```diff
@@ -208,6 +210,43 @@ export async function cli(args) {
 			break;
 		}
 
+		case 'badges': {
+			function showTemplateHelp() {
+				help({ command: 'help', args: ['badges'] });
+			}
+
+			if (!options.args.length) {
+				showTemplateHelp();
+				return;
+			}
+
+			const [command] = options.args;
+
+			switch (command) {
+				case 'init':
+					await initBadgeTemplate(options);
+					break;
+
+				case 'list':
+					await listBadgeTemplates(options);
+					break;
+
+				case 'archive':
```

Comment:
Maybe we should keep the output messaging consistent (don't care what it says so much as it would be good to keep it consistent)

<img width="431" alt="image" src="https://github.com/searchspring/snapfu/assets/5060400/2d1ef31f-69b3-4dad-9da3-2351c391802d">

^ also looks like an existing bug there with the `recs archive` output showing that "%" character.

Reply:
Looks like a ZSH thing. But I'm able to fix it. 
https://unix.stackexchange.com/questions/167582/why-zsh-ends-a-line-with-a-highlighted-percent-symbol